modules/noise/noise_texture.cpp

```diff
@@ -227,6 +263,26 @@ bool NoiseTexture::get_invert() const {
 	return invert;
 }
 
+void NoiseTexture::set_in_3d_space(bool p_enable) {
+	in_3d_space = p_enable;
```

Comment:
```suggestion
	if (p_enable == in_3d_space) {
		return;
	}
	in_3d_space = p_enable;
```
Same check as in most of the other setters to prevent regenerating the texture when the value is unchanged.